libs/entry-components/styles/modules/components/slide-togglers/_generator.scss

```diff
@@ -0,0 +1,33 @@
+/* stylelint-disable scss/selector-no-redundant-nesting-selector */
+@use 'sass:map';
+@mixin generate-from($theme) {
+	.mat-mdc-slide-toggle {
+		$enabled-states: 'enabled', 'enabled:focus:not(:active)', 'enabled:active', 'enabled:hover:not(:focus):not(:active)';
+
+		$track-on-background: map.get($theme, 'general', 'slide-togglers', 'on', 'track', 'background');
+		$track-off-background: map.get($theme, 'general', 'slide-togglers', 'off', 'track', 'background');
+		$handle-on-background: map.get($theme, 'general', 'slide-togglers', 'on', 'handle', 'background');
+		$handle-off-background: map.get($theme, 'general', 'slide-togglers', 'off', 'handle', 'background');
+
+		@each $state in $enabled-states {
+			.mdc-switch:#{$state} {
+				& .mdc-switch__track::after {
```

Comment:
unnecessary spacing

Reply:
It is done by purpose, It can't be separated, since mdc-switch__track is child element.